homeassistant/components/light/mqtt.py

```diff
@@ -87,6 +105,14 @@
     vol.Optional(CONF_RGB_STATE_TOPIC): mqtt.valid_subscribe_topic,
     vol.Optional(CONF_RGB_VALUE_TEMPLATE): cv.template,
     vol.Optional(CONF_STATE_VALUE_TEMPLATE): cv.template,
+    vol.Optional(CONF_HUE_COMMAND_TOPIC): mqtt.valid_publish_topic,
+    vol.Optional(CONF_HUE_STATE_TOPIC): mqtt.valid_subscribe_topic,
+    vol.Optional(CONF_HUE_VALUE_TEMPLATE): cv.template,
```

Comment:
Please also add a `CONF_HS_COMMAND_TEMPLATE`.

Reply:
@OttoWinter 
How to templates call?

 if ATTR_HS_COLOR in kwargs and \
           self._topic[CONF_HUE_COMMAND_TOPIC] is not None and \
           self._topic[CONF_SATURATION_COMMAND_TOPIC] is not None:

            hs_color = kwargs[ATTR_HS_COLOR]
            tpl = self._templates[CONF_HUE_COMMAND_TEMPLATE]
            if tpl:
                hue_value = tpl.async_render(hs_color[0])
            else:
                hue_value = int(hs_color[0])

            saturation_value = int(hs_color[1])


error:
 File "/usr/src/app/homeassistant/helpers/template.py", line 129, in async_render                                                     
    kwargs.update(variables)                                                                                                           
TypeError: 'float' object is not iterable

Reply:
Why do I need CONF_HS_COMMAND_TEMPLATE?

Added CONF_HUE_COMMAND_TEMPLATE.